.github/workflows/build-test.yaml

```diff
@@ -1,10 +1,6 @@
 ---
 name: "Build & Test"
 on: # yamllint disable-line rule:truthy
-  push:
-    branches:
-      - "!dependabot/*"
-      - "*"
```

Comment:
This was double-running all of our tests. If we still need to do something different with dependabot we can address that separately.